### ISSUE TITLE
Date/Time field is displaying UTC instead of local time.

### ISSUE THREAD

**Issue by tgspacelabs** [pods-framework/pods#5403]
**Describe the bug**

I use Pods to create a list of date/times for today's date to display a status value.  The display used to work perfectly for some time now.  Today I noticed that the time appears to be displaying UTC rather than local time.  

[https://www.skydivesnohomish.com/contact/tandem-class-status/](https://www.skydivesnohomish.com/contact/tandem-class-status/)

**To Reproduce**

**Expected behavior**
In the example page URL it used to display 12pm to 7:30pm and now appears to be UTC times which are +7 hours.

**Screenshots**
![image](https://user-images.githubusercontent.com/24533761/60455166-9a9ace80-9bea-11e9-8458-63ede2a21d81.png)

## Pods Version
Please provide the version of Pods you are running: 
Version: 2.7.13

## WordPress Environment

<details>

Copy and paste your System Details Tools, Site Health, Info. Use the 'Copy Site Info to Clipboard' and paste the results over this line.

</details>

## Pods Package Export (helpful!)

```
Copy and Paste the JSON Export from **Pods Admin, Migrate: Packages, Export** in WordPress admin here
```

**Additional context**

**Possible Workaround**
If you have discovered a workaround, please include it below.

**Comment by tgspacelabs** [comment]
Rolling back to Version 2.7.12 fixed the problem.

![image](https://user-images.githubusercontent.com/24533761/60455702-ffa2f400-9beb-11e9-823b-9462a2990856.png)

**Comment by pglewis** [comment]
Most likely the same as #5402, though this one probably should be the canonical issue because I do believe it's directly related to UTC.  This is reproduced locally and I'm looking into it.

**Comment by pglewis** [comment]
Appears to be introduced here: https://github.com/pods-framework/pods/pull/5297/files

**Comment by kengmick** [comment]
I believe this same problem in Pods Version 2.7.13 is showing up in admin and front end of a development site, delaying launch, https://www.mokshayoga.kenmick.com/ where I'm using local time DST Chicago = -5 UTC.  

Class start time of noon displays as 5pm, on edit displays as 5pm, can adjust to noon, and on save reverts to 5pm.  
see video:  https://cl.ly/eff91a75e298
I think this problem is causing multiple reports yet to be noted as duplicates, including:  

- this issue [Date/Time field is displaying UTC instead of local time. #5403](https://github.com/pods-framework/pods/issues/5403) status: Open; type: bug
- [BUG: Date/time field input is losing an hour on updating the post #5402](https://github.com/pods-framework/pods/issues/5402) status: Open; label: closed duplicate
- [UTC time issue on custom pods date time fields #5382](https://github.com/pods-framework/pods/issues/5382) status: Open; label: status: reproduced
- [problems with date #5407](https://github.com/pods-framework/pods/issues/5407) status: Open; label: type: bug
- and potentially [UTC time issue on custom pods date time fields #5382](https://github.com/pods-framework/pods/issues/5382) status: Open; status: reproduced

**Comment by pixelated-au** [comment]
@sc0ttkclark This issue should not be in backlog. It's a major flaw around times and renders Pods broken when it comes to times. Whenever we're saving, we're having to update every record to add 10 hours (Australia +10) before saving – otherwise, the time ends up being wrong. This is required **every-single-time** a record is saved irrespective of whether the date/time has been altered. Please change this issue to priority!

**Comment by pglewis** [comment]
#5409 is a candidate fix for this and if passes QA it will be released as a hot-fix ASAP.

**Comment by sc0ttkclark** [comment]
Solved by #5409
